add ability to pass timestamp

gregory opened the pull request.

skeggse commented on spec/src/indexSpec.js:
```diff
@@ -136,6 +136,41 @@ describe('cloudwatch-metrics', function() {
+        sendCallback: done,
+      });
+
+      metric.put(1, 'metricName', [{Name:'ExtraDimension',Value: 'Value'}]);
```
```suggestion
      metric.put(1, 'metricName', [{Name: 'ExtraDimension', Value: 'Value'}]);
```

skeggse commented on spec/src/indexSpec.js:
```diff
@@ -136,6 +136,41 @@ describe('cloudwatch-metrics', function() {
+            }],
+            MetricName: 'metricName',
+            Unit: 'Count',
+            Timestamp: jasmine.any(Number),
```
Could also verify that it's a number that's `>=` the value of `Date.now()` when the test case started

gregory commented:
@skeggse - addressed your feedbacks. 
As for the Travis setup duplicated in #24  - i thought those solve different issues. What about i rebase whichever PR get merged last?

gregory commented:
@skeggse  let me know what you think about the PR :)

skeggse commented on spec/src/indexSpec.js:
```diff
@@ -136,6 +136,42 @@ describe('cloudwatch-metrics', function() {
+          }],
+          Namespace: 'namespace'
+        });
+        expect(Date.parse(data.MetricData[0].Timestamp)).toBeLessThan(Date.now());
```
millisecond precision affords for this to be the same as the timestamp
```suggestion
        expect(Date.parse(data.MetricData[0].Timestamp)).toBeLessThanOrEqual(Date.now());
```
